lib/player.rb

```diff
@@ -0,0 +1,100 @@
+require_relative 'scoring'
+require_relative 'tile_bag'
+
+module Scrabble
+  class Player
+    attr_accessor :name, :tiles
+
+    def initialize(name)
+      @name  = name
+      @plays = []
+      @tiles = []
+    end
+
+    def tiles
+      if @tiles.length <= 7
+        return @tiles
+      else
+        raise ArgumentError.new("Cannot select more than 7 tiles")
```

Comment:
I like the idea of double checking this, but I'm not sure this is the right place. Generally, you should put checks and raise exceptions as soon as a problem might be created. As is, if someone were to rescue this exception, they would have no way to find out what the bad tiles are.